integ/test_extended_counters.py

```diff
@@ -28,7 +28,7 @@ def pytest_funcarg__servers(request):
     port = random.randrange(10000, 65000)
     config_path = os.path.join(tmpdir, "config.cfg")
     conf = """[statsite]
-flush_interval = 1
+flush_interval = 2
```

Comment:
This has been changed to be able to properly test the rate (with this being one the rate and the sum are equal)